### PR TITLE
Fix ansible-lint errors in wordpress-nginx/site.yml

### PR DESCRIPTION
**Pull request by ScottBrenner**
Fixing `ansible-lint` errors in `wordpress-nginx/site.yml` from https://github.com/ansible/ansible-examples/issues/271. With these changes:

```
$ ansible-lint wordpress-nginx/site.yml
<nothing>
```

Reference: https://docs.ansible.com/ansible-lint/rules/default_rules.html

_closes #271_

### CODE REVIEW

File: wordpress-nginx/roles/wordpress/tasks/main.yml [hunk 1]
```diff
@@ -13,10 +16,12 @@
   user: name=wordpress group=wordpress home=/srv/wordpress/
 
 - name: Fetch random salts for WordPress config
-  local_action: command curl https://api.wordpress.org/secret-key/1.1/salt/
+  get_url:
```
Comment (webknjaz) [inline wordpress-nginx/roles/wordpress/tasks/main.yml:19]: This should remain local

File: wordpress-nginx/roles/wordpress/tasks/main.yml [hunk 2]
```diff
@@ -13,10 +16,12 @@
...
+    url: https://api.wordpress.org/secret-key/1.1/salt/
   register: "wp_salt"
   become: no
   become_method: sudo
+  changed_when: false
```
Comment (webknjaz) [inline wordpress-nginx/roles/wordpress/tasks/main.yml:24]: Why?

File: wordpress-nginx/roles/wordpress/tasks/main.yml (hunk 1 above)
Comment (ScottBrenner) [reply wordpress-nginx/roles/wordpress/tasks/main.yml:19]: `ansible-lint` error here was
```
[303] curl used in place of get_url or uri module
wordpress-nginx/roles/wordpress/tasks/main.yml:15
Task/Handler: Fetch random salts for WordPress config
```

File: wordpress-nginx/roles/wordpress/tasks/main.yml (hunk 2 above)
Comment (ScottBrenner) [reply wordpress-nginx/roles/wordpress/tasks/main.yml:24]: `ansible-lint` error here was
```
[301] Commands should not change things if nothing needs doing
wordpress-nginx/roles/wordpress/tasks/main.yml:15
Task/Handler: Fetch random salts for WordPress config
```

File: wordpress-nginx/roles/wordpress/tasks/main.yml (hunk 1 above)
Comment (webknjaz) [reply wordpress-nginx/roles/wordpress/tasks/main.yml:19]: Exactly:
> [504] Do not use 'local_action', **use 'delegate_to: localhost'**

File: wordpress-nginx/roles/wordpress/tasks/main.yml (hunk 2 above)
Comment (webknjaz) [reply wordpress-nginx/roles/wordpress/tasks/main.yml:24]: I'd argue that it should be
```suggestion
  changed_when: true
```

File: wordpress-nginx/roles/wordpress/tasks/main.yml (hunk 1 above)
Comment (ScottBrenner) [reply wordpress-nginx/roles/wordpress/tasks/main.yml:19]: https://github.com/ansible/ansible-examples/pull/278/commits/fe001733534b7f6bf2d418c627c5bbb091d9584f

**Pull request merged**